src/core/ledger/policies/immediate.js

```diff
@@ -65,11 +67,22 @@ export const immediateConfigParser: P.Parser<ImmediatePolicy> = P.object({
   numIntervalsLookback: P.number,
 });
 
-export const immediatePolicyParser: P.Parser<ImmediatePolicy> = P.object({
-  policyType: P.exactly(["IMMEDIATE"]),
-  budget: grainParser,
-  numIntervalsLookback: P.number,
-});
+export const immediatePolicyParser: P.Parser<ImmediatePolicy> = P.fmap(
+  P.object(
+    {
+      policyType: P.exactly(["IMMEDIATE"]),
+      budget: grainParser,
+    },
+    {
+      numIntervalsLookback: P.number,
+    }
+  ),
+  (policy) => ({
+    ...policy,
+    numIntervalsLookback:
+      policy.numIntervalsLookback != null ? policy.numIntervalsLookback : 1,
+  })
```

Comment:
LGTM.  Nit: maybe kick this out into its own function so the parser's fmap can be more descriptive calling something like `upgradeLookback`.